application/views/js_init/js_layout.php

```diff
@@ -13,16 +13,32 @@ function new_notification(refreshmode) {
 			$('span.unread_inbox_notif').text(unreadcount[0]);
 			$('span.unread_spam_notif').text(unreadcount[2]);
 
-			var current_title = $(document).attr('title');
-			var stopNumber = current_title.search('\\)');
-			if (stopNumber != '-1') var title = current_title.substr(stopNumber + 1);
-			else var title = current_title;
-
-			var newtitle = unreadcount[0] + ' ' + title;
-			$(document).attr('title', newtitle);
+			// example of title: "(23) Kalkun"
+			var title = $(document).attr('title');
+			var re_title = title.match('(\\((\\d*)\\) )?(.*)');
+			var unreadcount_inbox_previous = re_title[2] ? re_title[2] : 0;
+			var title_cleaned = re_title[3];
+
+			var re = unreadcount[0].match('\\((.*)\\)');
+			var unreadcount_inbox_current = re[1] ? re[1] : 0;
+
+			var title_new = unreadcount[0] + ' ' + title_cleaned;
+			$(document).attr('title', title_new);
+
+			/*
+			console.debug(
+				"title: " + title +
+				"\ntitle_cleaned: " + title_cleaned +
+				"\nunreadcount_inbox_previous: " + unreadcount_inbox_previous +
+				"\nunreadcount_inbox_current: " + unreadcount_inbox_current +
+				"\ntitle_new: " + title_new
+			);
+			console.debug(re_title);
+			console.debug(re);
+			*/
```

Comment:
Instead of this, shouldn't triggering the sound where it fetches new message would make sense and remove the dependency of triggering based on title ?

Reply:
Maybe you could open a feature request for this or implement it ;) For now this avoids to have a sound played every minute...

Reply:
Opened #297